assets/app/features/upgrade/controllers/upgrade-upgrade-administration-server.controller.spec.js

```diff
@@ -65,6 +65,44 @@ describe('Upgrade Flow - Upgrade Administration Server Controller', function ()
         should.exist(controller);
     });
 
+    describe('on controller creation', function () {
+        afterEach(function () {
```

Comment:
Could this be a beforeEach() in a specific describe section for the last two cases?